src/model/signatureMatrix.js

```diff
@@ -20,8 +20,9 @@ for (let x = 0; x < COLUMNS; x++) {
     const isBonus = isBonusCell(cellNumber);
     const isSpecial = isSpecialCell(cellNumber);
 
-    const signature = signatures.unshift();
-    matrix.push({ isBonus, isSpecial, signature });
+    const signature = signatures.shift();
```

Comment:
I just made this same change and pushed straight to mastere. Sorry

Reply:
No worries man! Rules are meant to be broken!

Reply:
it's alright man, rules are made to be broken.

Reply:
Rules are made ot be broken

Reply:
YOU DID WAT?